tox.ini

```diff
@@ -1,12 +1,6 @@
 [tox]
 envlist = py27, py36, py37, flake8
 
-[travis]
```

Comment:
Perhaps this is not needed, but when I did this, I changed travis to gh-actions here (could just be a different way and both work, not sure).  See here:
https://github.com/release-depot/flask-container-scaffold/blob/main/tox.ini

Reply:
I tested the job against my branch while working on it and it seems to set up the versions correctly without it: https://github.com/jpichon/git_wrapper/actions/runs/574993890

Reply:
Then again, that might be why I had to specify -epy: just using "tox" as a command ran zero tests.

Thinking further, I'm tempted to leave it as is because it means updating the version matrix only requires a change in the workflow file, as opposed to remembering to update both test.yml and tox.ini?

Reply:
I can see that as a benefit, and have no issue with it - perhaps we will change the other projects to do the same.